src/univariate/continuous/lognormal.jl

```diff
@@ -62,6 +62,7 @@ stdlogx(d::LogNormal) = d.σ
 mean(d::LogNormal) = ((μ, σ) = params(d); exp(μ + σ^2/2))
 median(d::LogNormal) = exp(d.μ)
 mode(d::LogNormal) = ((μ, σ) = params(d); exp(μ - σ^2))
+partype(d::LogNormal{T}) where {T<:Real} = T
```

Comment:
The argument name is not needed (in fact, the language server will complain about it):
```suggestion
partype(::LogNormal{T}) where {T<:Real} = T
```